Assets/Resources/Shaders/TransparentEdgePortalShader.shader

```diff
@@ -0,0 +1,127 @@
+Shader "Unlit/SEE/TransparentEdgePortalShader"
+{
+    Properties
+    {
+        // Color
+        _Color("(Start) Color", color) = (1,0,0,1)
+        _EndColor("End Color", color) = (0,0,1,1)
+        _ColorGradientEnabled("Enable Color Gradient?", Range(0, 1)) = 0 // 0 = false, 1 = true
+
+        // Data Flow
+        _EdgeFlowEnabled("Enable Data Flow Visualization?", Range(0, 1)) = 0 // 0 = false, 1 = true
```

Comment:
The edges are initially created with a plain white color and later the gradient gets enabled -- after setting the target colors. This might even be visible for a few frames and makes it easier to distinguish between the states.

Reply:
I see, thanks for the explanation!